### PR TITLE
Added Radare2 Explorations post

### PR DESCRIPTION
**Pull request by monosource**

### CODE REVIEW

File: content/posts/radare2-explorations.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"radare2-explorations"
+]
++++
+by http://github.com/monosource
+
+# Context
+
+One of the challenges that people must face when starting to learn radare is getting to know how to use it for their specific needs; after all, not everyone uses radare for exploitation or forensics. As mentioned before, radare is indeed [documented](http://radare.today/posts/radare2-is-documented/).
+
+If the difficulty is not related to a lack of documentation, then where does it originate from? There's no middle ground between painstakingly experimenting with each command in radare and replicating the writeups that people post online for often difficult challenges they solved. Beginners will often find themselves stumbling between understanding how radare was used and the task itself.
```
Comment (radare) [inline content/posts/radare2-explorations.md]: painstakingly?

**Comment by monosource** [comment]
http://www.dictionary.com/browse/painstakingly

Want me to replace it?

File: content/posts/radare2-explorations.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+This was the motivation for writing [Radare2 Explorations](https://www.gitbook.com/book/monosource/radare2-explorations/details), a relatively short, free, open-source book which covers radare from a practical point of view, rather than a pedantic one.
+
+# The Book
+
+Its purpose is not to replace the already existing [Radare2 Book](https://www.gitbook.com/book/radare/radare2book/details), but rather to complement it.
+
+What does it cover? All the basics required to get you going, plus a few tutorials which are focused on how to solve very simple but specific tasks using radare.
+
+The introductory section is somewhat linear, although you can easily read the topics that interest you. The tutorial section is completely modular and each tutorial is a self-contained experience.
+
```
Comment (radare) [inline content/posts/radare2-explorations.md]: add comment (+ links) to download the PDF, ePUB, MOBI, i know those links are in gitbook, but will make it more visible for the reader and seo

Comment (monosource) [inline content/posts/radare2-explorations.md]: Do you want the links to be in the blog post or on the gitbook page (like the radare2 book)?

Comment (radare) [inline content/posts/radare2-explorations.md]: maybe just commenting that the book can be downloaded in those formats and make the works link to the gitbook page

**Comment by monosource** [comment]
Looks good to me. Let me know if you want any other modifications :)

**Comment by radare** [comment]
lgtm :)

**Pull request merged**